pyproject.toml

```
@@ -1,12 +1,34 @@
-[build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
-build-backend = "setuptools.build_meta"
+[project]
+name = "cvxopt"
+dynamic = ["version"]
+description = "Convex optimization package"
+authors = [
+    {name = "M. Andersen", email = "martin.skovgaard.andersen@gmail.com"},
+    {name = "J. Dahl", email = "dahl.joachim@gmail.com"},
+    {name = "L. Vandenberghe", email = "vandenbe@ee.ucla.edu"}
+]
+dependencies = []
+requires-python = ">=3, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
```

I'm not fluent in the dependency mini-language, but couldn't this be written
```suggestion
requires-python = ">=3.5"
```